doc/classes/@GlobalScope.xml

```diff
@@ -702,7 +701,7 @@
 				nearest_po2(0)  # Returns 0 (this may not be expected)
 				nearest_po2(-1) # Returns 0 (this may not be expected)
 				[/codeblock]
-				[b]Warning:[/b] Due to the way it is implemented, this function returns [code]0[/code] rather than [code]1[/code] for negative values of [param value] (in reality, 1 is the smallest integer power of 2).
+				[b]Warning:[/b] Due to its implementation, this method returns [code]0[/code] rather than [code]1[/code] for negative values of [param value]. It also has no effect if [param value] is at the negative integer limit.
```

Comment:
- As usual, I dislike "negative" being used as "less than or equal to zero". For me negative by default means "less than zero". What I mean is it's potentially not clear so rewording might make sense.
- "values of value" might confuse some people.
- Last sentence is not clear, what does it mean "also has no effect"? Also like what? What else has no effect?

Maybe something like this:
```suggestion
				[b]Warning:[/b] Due to its implementation, this method returns [code]0[/code] rather than [code]1[/code] for values less than or equal to [code]0[/code], with an exception for [param value] being the smallest negative 64-bit integer ([code]-9223372036854775808[/code]) in which case the `[param value]` is returned unchanged.
```
?

Reply:
You're right, this is clearer.